### ISSUE TITLE
C-ICAP Container Pool

### ISSUE THREAD

**Issue by BigPeeg** [filetrust/icap-performance-tests#23]
The current pool of ICAP Endpoints (in the  icap-pilot-resource resource group) are using ICAP Containers hosted from a personal docker hub. A new set of endpoints should be made available that are provided by containers generated from the image stored by the c-icap repo into an Azure Container Registry.

Create a new resource group in the ICAP Subscription, gw-icap-performance-pool.
Within this RG deploy 5 Container Instances
- Each container needs to expose port 1344
- Each container instance should have 1 CPU and 1.5 GiB of Memory, with no GPU.
- The container is accessible from 
  - REPOSITORY: glasswallicap.azurecr.io
  - IMAGE: icap-test
  - RESOURCE_GROUP: gw-gks-icap-squid-proxy

The deployment of these containers should be scripted, to enable easy replacement in the event of the C-ICAP image (icap-test) being updated. These scripts should be stored in the 'icap-performance-tests' repo with supporting documentation.

**Comment by mpigram** [comment]
Looking to automate the deployment of this using Palumi. Will document everything and create a read me on how to use the deployment stack.

**Comment by mpigram** [comment]
Pulumi deployment all completed, I'm just waiting for IT to give me access to the icap azure subscription then I can make sure it's pulling in the correct images.

**Comment by mpigram** [comment]
I've created Pulumi scripts for the following:

- Registry creation
- Container creation

There is currently a new container registry "glasswallicap58de6aa8" within the resource group "gw-icap-performance-pool-c11c167a". I've also created 5 separate container instances all running the latest image of the icap-test.

I'm about to write out my documentation and then create a PR but wondered if we could get someone from the Dev team to test the endpoints?

Thanks,
Matt
